test/containers/HabitBox.spec.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import HabitBox from '../../client/components/HabitBox';
+
+describe('(Container) HabitBox', () => {
+  const wrapper = shallow(<HabitBox />);
+
+  it('renders as a <div>', () => {
+    expect(wrapper.type()).to.eql('div');
+  });
+
+  it('has style with height 100%', () => {
+    const expectedStyles = {
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).